src/main/java/org/jitsi/videobridge/health/Health.java

```diff
@@ -88,7 +88,7 @@ private static void check(Conference conference)
                 throw new RuntimeException(ioe);
             }
 
-            endpoints.add(endpoint);
+            //endpoints.add(endpoint);
```

Comment:
What's this?

Reply:
There's a block of code below which uses `endpoints`, but it's commented out.

Reply:
So nothing is using `endpoints`?

Reply:
Ah, ok...I expanded the file and see now.